python3.10/README.md

```diff
@@ -0,0 +1,136 @@
+Python 3.10.0 static compilation
+
+Prerquisites:
+
+Ubuntu 18.04 VM with 4Gb RAM 2 VCPU
+
+configuration stored in vm_dumpxml.xml file
+
+uname -a output:
+
+Linux ubuntu18-buildserver 4.15.0-88-generic #88-Ubuntu SMP Tue Feb 11 20:11:34 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
+
+
+
+
+Build preparation process:
+
+Install needed packages from file vm_installed_packages.txt
+
+while read -r line; do sudo apt-get -y install "$line"; done < vm_installed_packages.txt
+
+
+
+
+Phase I install static compilator musl:
+
+
+
+export MUSL_PATH=/root/build/musl
+
+mkdir $MUSL_PATH
+
+tar -zxvf musl-1.2.2.tar.gz
+
+cd musl-1.2.2/
+
+./configure --prefix=$MUSL_PATH
+
+make
+
+make install
+
+export CC=/root/build/musl/bin/musl-gcc
+
+	RESULT $MUSL_PATH/bin/musl-gcc
+
+
+
+
+Phase II build zlib static lib
+
+
+
+tar -xvf zlib-1.2.11.tar.xz
+
+cd zlib-1.2.11/
+
+CFLAGS="-static -fPIC" ./configure --static
+
+make
+
+	RESULT: libz.a
+
+
+
+
+Phase III build OpenSSL static lib
+
+
+
+mkdir /usr/local/ssl
+
+tar -zxvf openssl-1.1.1l.tar.gz
+
+CFLAGS="-static -idirafter /usr/include/ -idirafter /usr/include/x86_64-linux-gnu/" ./config no-shared --static --prefix=/usr/local/ssl1
+
+	This tip was got from https://github.com/openssl/openssl/issues/7207
+
+make
+
+make install
+
+	RESULT: /usr/local/ssl1/lib/libssl.a
+
+
+
+TIP: To choose required ssl version:
+
+	Download python 3.10
```

Comment:
add python installation command apt ....

Reply:
Default apt repo doesn't contain last python. 
python3.10 uses openssl-1.1.1l.
This tip is for future builds with next versions of python when openssl version will be unknown again.